### PR TITLE
[UI] Index view breadcrumb #2662

### LINKED ISSUE #2662: [UI] Index view breadcrumb
**To Reproduce**

1. Go to https://community-tc.services.mozilla.com/tasks/index
2. Type anything in the search input field (e.g., "foo")

**Expected behavior**
Text should only change in the input field. The breadcrumb view should not be changed until the user presses enter.

**Actual behavior**
The breadcrumb view is being displayed and the text changes while the search input field is changing.

Screen Shot 2020-04-14 at 1 38 13 PM

**Taskcluster version**
v29.0.1

### PR DESCRIPTION
**Pull request by aryamanpuri**
Github Bug/Issue: Fixes #2662

### CODE REVIEW

File: changelog/issue-2662.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+audience: deployers
+level: minor
```
Comment (djmitche) [inline changelog/issue-2662.md:2]: There's some advice on changelog levels [here](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md).  In this case, I suspect `silent` is the right choice, unless this is fixing an issue that a user has identified and will want to see fixed.

Comment (aryamanpuri) [reply changelog/issue-2662.md]: Done

**Comment by helfi92** [comment]
@aryamanpuri Did you test that this works?

**Comment by helfi92** [comment]
@aryamanpuri Let me know if you have any questions or whether you are having a hard time setting up the DB. Happy to help.

**Comment by aryamanpuri** [comment]
Thanks
I was getting some errors on `yarn start`.
```
ERROR in ./src/components/SchemaTable/index.jsx
Module not found: Error: Can't resolve 'material-ui-json-schema-viewer/build/SchemaViewer' in '/home/aryaman/taskcluster/ui/src/components/SchemaTable'

ERROR in ./src/components/DiffTextArea/index.jsx
Module not found: Error: Can't resolve 'react-gh-like-diff' in '/home/aryaman/taskcluster/ui/src/components/DiffTextArea'

ERROR in ./src/components/DiffTextArea/index.jsx
Module not found: Error: Can't resolve 'react-gh-like-diff/dist/css/diff2html.min.css' in '/home/aryaman/taskcluster/ui/src/components/DiffTextArea'

ERROR in ./src/views/Tasks/TaskIndex/IndexedTask/index.jsx 152:35
Module parse failed: Unexpected token (152:35)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
 * ./node_modules/eslint-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
|     } = this.props;
|     const loading = latestArtifactsLoading || indexedTaskLoading;
>     const indexPaths = indexedTask?.namespace?.split('.') ?? [];
|     return React.createElement(Dashboard, {
|       title: "Index Browser",

ERROR in ./src/views/Provisioners/ViewWorkers/index.jsx 397:19
Module parse failed: Unexpected token (397:19)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
 * ./node_modules/eslint-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
|       },
|       __self: this
>     }), workerType?.actions?.length ? React.createElement(SpeedDial, {
|       __source: {
|         fileName: _jsxFileName,

```

**Comment by helfi92** [comment]
Looks like some dependencies are not properly installed or out of date? Try deleting the node_modules directory in that's inside the `ui` directory (`cd ui && rm -rf node_modules/`) and running `yarn` again to re-install them. Does that help?

**Comment by aryamanpuri** [comment]
Closing this for now

**Pull request closed without merging**